### ISSUE TITLE
Some suggestions~

### ISSUE THREAD

**Issue by akaAMAZING** [cpieprzak/PoeAggregator#28]
First of all, amazing amazing tool - I have been using it for a while now and love it!

I have come up with a couple of ideas (they may not be completely viable.. that is up to you lol):

1. Periodically go through the "offers sent" list, refresh the listing and message again if the listing is still available. I do this manually sometimes from time to time and find that I get a majority of items that I re-message for. Sometimes the instant message scares the seller and they don't want to sell right away. Other times the seller is busy mapping (as the item is usually force listed due to a instance change) and they are not willing to leave it right away.
2. Lots of sellers are usually offline from the time of searching for an item. Is it possible to include people that come online during the search? Basically as though including people that were offline at the time of searching as new sellers as they come online.

Please let me know if I can help.

**Comment by akaAMAZING** [comment]
Oh another one:

3. If you are searching for multiple listings and a seller lists multiple items that you're interested in for sale, the program currency spams all the trade requests in 0.5 seconds lol. 

For example, earlier I was searching for 4 different fossils. One user listed all 4 fossils for sale and then got overwhelmed I guess by the number of instant PM's that he never sells.

Is it possible to set a delay before PM-ing the same person? For example 5 seconds or something?

Thank you!

**Comment by cpieprzak** [comment]
:)
#1 Interesting idea.  I would have to think about how you could do this without being intrusive to playing the game.  I wouldn't want it to just auto message like 30 seconds later... cuz I might be messaging someone or something. Perhaps just popup the item in the live search window again if you messaged and it is still there.  Not sure, let me know thoughts.


#2 Already exists.  If you click watch item on an offline item, when that person comes online (or the price changes) you will be notified.

**Comment by akaAMAZING** [comment]
#1 Yes that would be amazing. Having the item appear again if it's still available after a given amount of time that can be specified by the user? But also we would have to watch for the rate limit too for refreshing items to see if they're still available right? Or how about implementing a "Auto Refresh" checkbox that would refresh a given listing that has priority. It will refresh it periodically given the time set by the user and if it still exists, have it appear at the top of the list ready to whisper as though it's newly listed.

#2 Yeah it exists for certain bespoke items but what if you are searching for a div card or some generic item where the price has been inflated due to people being offline. For example: Many people have listed item X for 30c in the last day. However, as they are all offline, item X is selling for 35-40c instead because people price on what is currently online. Could it be made so that when an offline listing comes online, then we get notified? So in the example, as soon as one of the people that have it listed for 30c comes online, we get an alert as though it's a new listing.

**Comment by akaAMAZING** [comment]
Continuing on #2 I'm curious as to how you add an item from the trade website onto the watchlist.

For example: https://prnt.sc/MDoPyU79Qa09

Not sure how to watch the item listed by rAiLRiva 🤔

**Comment by cpieprzak** [comment]
Ah so to watch an item you have to get it into the main search window.
![image](https://user-images.githubusercontent.com/49933714/158509551-85f0aabf-c3b3-4ced-af04-6ce79aa1554e.png)

**Comment by akaAMAZING** [comment]
Yeah can it be made so that we can 'watch' any item? Not just one that has passed through the filter?

Even so, the watch feature doesn't really tackle #2 but I'm not sure anything does 🤔

**Comment by cpieprzak** [comment]
Not without a very large amount of effort.  More than I'm willing to do at the moment.
If your asking for a live search for offline to online for a item query, that's not possible.

**Comment by akaAMAZING** [comment]
Yeah I assumed it wouldn't be possible as there's no way to know unless we page refresh.

Thank you for your continued efforts though <3 this is without doubt the best POE search manager I've seen.

**Comment by otakumaniac666** [comment]
Hi is live searching possible with this tool if so how?? i cant seem to get it to work
